src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/policy.py

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import LongRunningOperation
+from ._utils import validate_managed_registry
+
+
+POLICIES_NOT_SUPPORTED = 'Policies are only supported for managed registries.'
+
+
+def acr_config_content_trust_show(cmd,
+                                  client,
+                                  registry_name,
+                                  resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, POLICIES_NOT_SUPPORTED)
+    return client.list_policies(resource_group_name, registry_name).trust_policy
+
+
+def acr_config_content_trust_update(cmd,
+                                    client,
+                                    registry_name,
+                                    status,
+                                    resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, POLICIES_NOT_SUPPORTED)
+
+    trust_policy = client.list_policies(resource_group_name, registry_name).trust_policy
+    if trust_policy.status != status:
```

Comment:
Can we remove this client side logic? Basically as long as `status` is not `None`, we should send the request and let the server decide what the updated policy is.